### PR TITLE
Fix/3008/Adjust links to docs

### PR DESCRIPTION
**Pull request by Hall-Ma**
# Adjust links to docs

closes: #3008

## Description
- links to new doc pages about sonarqube were broken and and have been fixed now

### CODE REVIEW

**Review by BridgeAR — APPROVED** [review]
RSLGTM

File: gh-pages/_docs/05-05-analyze-with-sonarqube.md [hunk 1]
```diff
@@ -1,9 +1,5 @@
 ---
-categories:
-    - How-To-SonarQube
-tags:
-    - sonarimport
-    - analysis
```
Comment (MW-Friedrich) [inline gh-pages/_docs/05-05-analyze-with-sonarqube.md:6]: Why was this removed, just curious
